model/version.go

```diff
@@ -13,6 +13,7 @@ import (
 // It should be maintained in chronological order with most current
 // release at the front of the list.
 var versions = []string{
+	"5.34.1",
```

Comment:
@grundleborg this is something that y'all agree?

Reply:
Is this because there's a dot release fixing a migration? If so, there's precedent for doing it this way so fine with me.

Reply:
when this was created was no plan to have a dot release.
this PR was generated by the "new pipeline" after we release the latest version in this case 5.34.0

but I don't know if the process is correct, after we release the main release, open a PR to bump this to the patch right away.

what I would like to know if the process is correct.

Reply:
Thing is the fixed migration is part of a previous version (5.32) and it will run based on some conditions (we essentially check if we need to do it, no matter what the db schema version is). So I am thinking this change might not be needed.

Reply:
ok @streamer45 it is your call, feel free to close or merge this PR

Reply:
one follow-up question, and this automatic PR is needed? or we will do that on-demand?

Reply:
I'll close it. I am thinking the automated PR makes sense for minor version (non dot) upgrades. But I'd defer to @metanerd for the specifics.

Reply:
specifics of what? does this was agreed to we have an automatic PR to bump this when we release? 


cc @jwilander FYI

Reply:
This is due to users getting confused by version differences. So if you update patch version of mattermost and not patch version DB, users are confused why it wasn't updated and if it is correct. I would keep it if it doesn't hurt, but of course it is not my decision.

Reply:
@metanerd @cpanato It is needed because then the server reports a wrong version, leading to issues like this https://github.com/mattermost/mattermost-server/issues/17436.